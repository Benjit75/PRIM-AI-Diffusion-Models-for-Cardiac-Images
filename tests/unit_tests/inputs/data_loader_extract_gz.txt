training/patient001/patient001_frame01.nii.gz
training/patient001/patient001_frame01_gt.nii.gz
training/patient001/patient001_frame12.nii.gz
training/patient001/patient001_frame12_gt.nii.gz
testing/patient101/patient101_frame01.nii.gz
testing/patient101/patient101_frame01_gt.nii.gz
testing/patient101/patient101_frame14.nii.gz
testing/patient101/patient101_frame14_gt.nii.gz