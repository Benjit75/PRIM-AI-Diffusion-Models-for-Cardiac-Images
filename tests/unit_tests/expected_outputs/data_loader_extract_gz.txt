training/patient001/patient001_frame01.nii
training/patient001/patient001_frame01_gt.nii
training/patient001/patient001_frame12.nii
training/patient001/patient001_frame12_gt.nii
testing/patient101/patient101_frame01.nii
testing/patient101/patient101_frame01_gt.nii
testing/patient101/patient101_frame14.nii
testing/patient101/patient101_frame14_gt.nii